spec/controllers/sites_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to specify the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator.  If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails.  There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+#
+# Compared to earlier versions of this generator, there is very limited use of
+# stubs and message expectations in this spec.  Stubs are only used when there
+# is no simpler way to get a handle on the object needed for the example.
+# Message expectations are only used when there is no simpler way to specify
+# that an instance is receiving a specific message.
+
+RSpec.describe SitesController, type: :controller do
+  # This should return the minimal set of attributes required to create a valid
+  # Site. As you add validations to Site, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) do
+    skip("Add a hash of attributes valid for your model")
```

Comment:
Can we fill `valid_attributes` and `invalid_attributes` out? That'll get the scaffolding tests up and running.